src/Network/XMPP/Types.hs

```diff
@@ -15,25 +15,13 @@
 -- Module      :  Network.XMPP.Types
 -- Copyright   :  (c) Dmitry Astapov, 2006 ; pierre, 2007
 -- License     :  BSD-style (see the file LICENSE)
--- 
+--
 -- Maintainer  :  Dmitry Astapov <dastapov@gmail.com>, pierre <k.pierre.k@gmail.com>
 -- Stability   :  experimental
 -- Portability :  portable
 --
 -----------------------------------------------------------------------------
-module Network.XMPP.Types
-( Server, Username, Password, Resource
-, XmppMonad, runXmppMonad, runXmppMonad'
-, Stream(..), StreamType(..)
-, Stanza(..), StanzaType(..), SomeStanza(..)
-, MessageType(..), PresenceType(..), IQType(..), ShowType(..)
-, RosterItem(..)
-, JID(..), JIDQualification(..), SomeJID(..), NodeID(..), ResourceID(..), DomainID(..)
-, StanzaPurpose(..)
-, Sing(..), IncomingSym0, OutgoingSym0, SStanzaPurpose
```

Comment:
Some of the exports will be different from the what `singletons` produce. So need to check if we still have enough generated stuff.

Reply:
I did the same, I don't think it makes a lot of sense to add these tight exports to a types module, (unless we want to hide some?)